scripts/pylint_custom_plugin/tests/test_pylint_custom_plugins.py

```diff
@@ -2678,6 +2768,14 @@ def test_disable_pylint_alias(self):
         with self.assertNoMessages():
             self.checker.visit_module(node)
 
+    def test_guidelines_link_active(self):
+        url = "https://github.com/Azure/autorest/blob/main/docs/generate/built-in-directives.md"
```

Comment:
do we actually include this link in the linting error?

Reply:
Nope, I will add it!

Reply:
C4745: Do not alias generated code. This messes up sphinx, intellisense, and apiview, so please modify the name of the generated code through the swagger / directives, or code customizations. See Details: https://github.com/Azure/autorest/blob/main/docs/generate/built-in-directives.md (naming-mismatch)